common/beerocks/bwl/dwpal/base_wlan_hal_dwpal.cpp

```diff
@@ -880,10 +930,16 @@ bool base_wlan_hal_dwpal::refresh_vap_info(int vap_id)
 
     vapElement.mac  = std::string(bssid);
     vapElement.ssid = std::string(ssid);
+    if (!get_vap_type(ifname, vapElement.fronthaul, vapElement.backhaul)) {
+        LOG(ERROR) << "Failed to get VAP type";
+        return false;
+    }
 
     // Store the VAP element
     LOG(DEBUG) << "Detected VAP ID (" << vap_id << ") - MAC: " << vapElement.mac
-               << ", SSID: " << vapElement.ssid;
+               << ", SSID: " << vapElement.ssid
+               << ", fronthaul: " << beerocks::string_utils::bool_str(vapElement.fronthaul)
+               << ", backhaul: " << beerocks::string_utils::bool_str(vapElement.backhaul);
```

Comment:
Why not use [`std::boolalpha`](https://en.cppreference.com/w/cpp/io/manip/boolalpha) instead?
```suggestion
               << ", fronthaul: " << std::boolalpha << vapElement.fronthaul
               << ", backhaul: " << vapElement.backhaul;
```